cineast-api/src/main/java/org/vitrivr/cineast/api/rest/handlers/actions/StatusInvokationHandler.java

```diff
@@ -6,6 +6,8 @@
 import io.javalin.plugin.openapi.annotations.OpenApi;
```

Comment:
While we're at it, this class should be called Statusinvo**c**ationhandler (to invoke something is an invocation with a c)

Reply:
It appears that this typo was there since the very beginning - it was fixed now